src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
I've been reading up on Mongo extended JSON http://docs.mongodb.org/manual/reference/mongodb-extended-json/ - and it seems to me that's what we should be supporting (we can start with just date support, and do the rest later on).

According to this, dates should normally be provided as an ISO-8601 string:

{ "$date": "<date>" }

OR, if the date is before the epoch start a long can be used:

{ "$date" : { "$numberLong" : "<dateAsMilliseconds>" } }